packages/helm-charts/blockscout/templates/grafana-hook.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}"
+  annotations:
+    "helm.sh/hook": post-upgrade
+spec:
+  template:
+    spec:
+      serviceAccountName: {{ .Release.Name }}-rbac
+      initContainers:
+{{ include "celo.blockscout.initContainer.secrets-init" . | indent 10 }}
+      containers:
+      - name: post-install-grafana
+        image: eruizgar91/gcloud-jq:latest
```

Comment:
We can't use your private docker image for that, can we build and host in in our registry?